### PR TITLE
 Add experimental Debian8 based Bazel container.

### PR DESCRIPTION
**Pull request by xingao267**

### CODE REVIEW

File: container/debian8/layers/bazel/BUILD [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+load(
+    "//container/common/bazel:version.bzl",
+    "BAZEL_VERSION_SHA256S",
+    "BAZEL_VERSION_TO_CONFIG_VERSION",
+)
+
+exports_files(glob(["*.yaml"]))
+
+# Used in substitution in bazel.yaml.tpl and bazel.bazelrc.tpl
+CONFIG_BASE = "debian8_clang/0.3.0"
```
Comment (nlopezgi) [inline container/debian8/layers/bazel/BUILD]: Can we put this CONFIG_BASE in a bzl file further up in the directory structure (maybe in https://github.com/bazelbuild/bazel-toolchains/blob/master/container/debian8/BUILD or in https://github.com/bazelbuild/bazel-toolchains/blob/master/configs/debian8_clang/BUILD)

File: container/ubuntu16_04/layers/bazel/BUILD [hunk 2]
```diff
@@ -20,13 +20,16 @@ load("//container/rules:docker_toolchains.bzl", "language_tool_layer")
...
-    "//container/ubuntu16_04/layers/bazel:version.bzl",
+    "//container/common/bazel:version.bzl",
     "BAZEL_VERSION_SHA256S",
     "BAZEL_VERSION_TO_CONFIG_VERSION",
 )
 
 exports_files(glob(["*.yaml"]))
 
+# Used in substitution in bazel.yaml.tpl and bazel.bazelrc.tpl
+CONFIG_BASE = "ubuntu16_04_clang/1.0"
```
Comment (nlopezgi) [inline container/ubuntu16_04/layers/bazel/BUILD]: same, move further up the directory structure

File: container/debian8/layers/bazel/BUILD (hunk 1 above)
Comment (xingao267) [reply container/debian8/layers/bazel/BUILD]: Done.

File: container/ubuntu16_04/layers/bazel/BUILD (hunk 2 above)
Comment (xingao267) [reply container/ubuntu16_04/layers/bazel/BUILD]: Done.

**Pull request merged**